.github/workflows/node-ci-pr.yml

```diff
@@ -0,0 +1,92 @@
+name: Node CI (PR)
```

Comment:
I copied this file from wonder-blocks and removed the parts we don't need at the moment.